src/routes/Chat/component.js

```diff
@@ -165,6 +166,9 @@ export default class Chat extends Component {
 							autoFocus={true}
 						/>}
 					</div>
+					<div>
+						{(messages[messages.length - 1] ? messages[messages.length - 1].t === 'jitsi_call_started' : messages[messages.length - 1]) ? <CallNotification rid={(messages[messages.length - 1])} /> : null}
```

Comment:
This is temporary right? I was under the impression that we'd be relying on the video call event instead of the message type to identify if the call has started